source/usage-examples/changeStream.txt

```diff
@@ -88,17 +88,17 @@ print change events that occur on the collection.
 
 First, open the change stream on the collection and then define a callback
 on the change stream using the ``on()`` method. Once set, generate a change
-event to be emitted by performing a change to the collection.
+event to emit by performing a change to the collection.
 
 To generate the change event on the collection, let's use ``insertOne()``
 method to add a new document. Since the ``insertOne()`` may run before the
-listener function can register, we use a timer, declared with ``setTimeout()``
-to wait one second before executing the insert.
+listener function can register, we use a timer, declared with
+``simulateAsyncPause`` to wait 100 milliseconds before executing the insert.
```

Comment:
```suggestion
``simulateAsyncPause`` to wait 1 second before executing the insert.
```
Suggestion: I think seconds is clearer than milliseconds (I don't intuitively know how long 100 milliseconds is). 1 second only works though if the time interval is raised.